packages/opencensus-core/src/stats/types.ts

```diff
@@ -176,16 +176,14 @@ export interface DistributionData extends AggregationData {
   sumSquaredDeviations: number;
   /** Bucket distribution of the histogram */
   buckets: Bucket[];
-  /** The bucket boundaries for a histogram */
-  readonly bucketsBoundaries: number[];
 }
 
 /** A simple histogram bucket interface. */
 export interface Bucket {
   /** Number of occurrences in the domain */
   count: number;
   /** The maximum bucket limit in domain */
-  readonly max: number;
+  readonly highBoundary: number;
```

Comment:
For the comments here, can we say "the maximum (min) possible value for a data point to fall in this bucket"?

Reply:
Sure, I liked it.